src/structures/guild.ts

```diff
@@ -28,7 +29,7 @@ export class Guild extends Base {
   verificationLevel?: string
   defaultMessageNotifications?: string
   explicitContentFilter?: string
-  roles?: Role[]
+  roles: RolesManager = new RolesManager(this.client, this)
   emojis?: Emoji[]
```

Comment:
This should be an `EmojiManager`

Reply:
members and channels properties too

Reply:
These properties are yet to be implemented using "Child Manager" classes which I'll be making using future PRs.

Reply:
okay